### PR TITLE
Error page separation

### PR DESCRIPTION
**Pull request by jonom**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+addons:
+  apt:
+    packages:
+      - tidy
+
+env:
+  global:
+    - TRAVIS_NODE_VERSION="6"
+    - COMPOSER_ROOT_VERSION="4.0.x-dev"
```
Comment (tractorcow) [inline .travis.yml]: errorpage is 1.0.x-dev not 4.0.x-dev

**Comment by chillu** [comment]
Verified tests passed locally on cms fork, and fixed composer root in `.travis.yml` (from 4.x to 1.x)

**Comment by chillu** [comment]
@jonom Great job! :) I'll add you as a package maintainer on packagist if you provide me your username there (`jonom` didn't work)

**Comment by jonom** [comment]
Thanks for reviewing @tractorcow and @chillu :)

> I'll add you as a package maintainer on packagist if you provide me your username there

@chillu Thanks but I'm not interested in maintaining this module going forward, I'm not a fan of the approach. I'll probably cook up a simpler alternative soonish.

**Comment by tractorcow** [comment]
That's part of being a core committer @jonom :D

**Pull request merged**